README.md

```diff
@@ -131,21 +131,6 @@ When elements are extracted from PDFs or images, it may be useful to get their b
   | jq -C . | less -R
 ```
 
-#### PDF Table Extraction
-
-To extract the table structure from PDF files using the `hi_res` strategy, ensure that the `pdf_infer_table_structure` parameter is set to `true`. This setting includes the table's text content in the response. By default, this parameter is set to `false` to avoid the expensive reading process.
-
-```
- curl -X 'POST' \
-  'https://api.unstructured.io/general/v0/general' \
-  -H 'accept: application/json' \
-  -H 'Content-Type: multipart/form-data' \
-  -F 'files=@sample-docs/layout-parser-paper.pdf' \
-  -F 'strategy=hi_res' \
-  -F 'pdf_infer_table_structure=true' \
-  | jq -C . | less -R
-```
-
 #### Skip Table Extraction
 
 Currently, we provide support for enabling and disabling table extraction for file types other than PDF files. Set parameter `skip_infer_table_types` to specify the document types that you want to skip table extraction with. By default, we skip table extraction
```

Comment:
here `By default, we skip table extraction`.... i realize that the default here will be change to perform table extraction for all file types (i assume this is the case?) Could you update it as well? thx

Reply:
ohhh right sorry i was confused

Reply:
so default is to skip every file expect pdf for hi res table extraction?

Reply:
that used to be the default, but let's change it to not skip any file types (empty list value for `skip_infer_table_types`). the cost for table extraction is not too high.

moving forward, the preferred way to skip table extraction for pdfs, png, and jpg (which adds a little of compute cost) would be:
skip_infer_table_types=["png","jpg","pdf"]

^^ the values to skip_infer_table_types should be correct json per https://github.com/Unstructured-IO/unstructured-api/pull/368

Reply:
@ds-filipknefel ok as this is a really small change - please remove the contents of the default list here:
https://github.com/Unstructured-IO/unstructured-api/blob/main/prepline_general/api/models/form_params.py#L75
and from the parameter description. 

Only leave `"['pdf', 'jpg', 'png']"` in the example param.
If any readmes need to be updated - please do it as well.